dotnet_sdk/lib/dependabot/dotnet_sdk/update_checker/latest_version_finder.rb

```diff
@@ -0,0 +1,140 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "excon"
+require "sorbet-runtime"
+
+require "dependabot/dotnet_sdk/requirement"
+require "dependabot/dotnet_sdk/version"
+require "dependabot/registry_client"
+require "dependabot/update_checkers/base"
+
+module Dependabot
+  module DotnetSdk
+    class UpdateChecker < Dependabot::UpdateCheckers::Base
+      class LatestVersionFinder
+        extend T::Sig
+
+        RELEASES_INDEX_URL = "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json"
```

Comment:
`@richlander` might be interested to know if you're going to depend on the release index JSON so he can work with you folks if/when they want to make any schema changes.

Reply:
Thanks, I'll reach out. I already discovered that the JSON they publish doesn't conform to the JSON schema that they also publish.

Reply:
https://github.com/SchemaStore/schemastore/pull/4135

Reply:
https://github.com/dotnet/core/issues/9541